python/lsst/cp/pipe/utils.py

```diff
@@ -374,15 +386,19 @@ def fitBootstrap(initialParams, dataX, dataY, function, confidenceSigma=1.):
     reducedChiSqBootstrap : `float`
         Reduced chi squared.
```

Comment:
Maybe amend this too so it's in keeping with the other one about the optional weighting? Or make the other like this, I think that's actually fine too.